dynamicweb/settings/base.py

```diff
@@ -186,7 +188,11 @@ def env(env_name):
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'app'
+        'NAME': 'app',
+        'HOST': 'localhost',
+        'USER': 'ubuntu',
+        'PASSWORD': 'Qwerty123',
+
```

Comment:
I think that's your local configurations to the database. Be careful, we don't want that.